### ISSUE TITLE
Improve redistricting calculation

### ISSUE THREAD

**Issue by j3camero** [j3camero/canada-election-forecast#13]
Currently the 2013 redistricting is handled by calculating the projections for the old ridings then taking a linear weightings of those results according to the total population overlap between the new and old ridings. For example, the projection for a new riding composed of 40% old riding A and 60% old riding B is 0.4_A + 0.6_B.

But I can do better. In addition to the total population "movement" between the old ridings and new ridings, I've got a record of which actual polls moved. This could help bigtime if an old riding was split along a natural party support boundary.

**Comment by j3camero** [comment]
Follow up with Heather when this is done.

**Comment by KelseyDH** [comment]
On wikipedia there is already quite a lot of data on most ridings showing redistributed election results based on poll.  Couldn't that just be used?

**Comment by j3camero** [comment]
True that. This problem is already fixed for those ridings that have local riding polls on Wikipedia. The others are still using the imprecise weighted mix of old ridings method. This is so quick to fix, I've really just gotta sit down and bang it out. Tomorrow.

**Comment by KelseyDH** [comment]
Yeah looking into the wikipedia redistributed results numbers they appear to come from Pundits Guide.  Its a little unclear whether they use a percentage of the redistributed results or redistribute based on geographic polling results.  If you have the results redistributed by geographic polling results it would be interesting to see them.
